map/src/test/java/openlr/map/simplemockdb/SimpleMockedLine.java

```diff
@@ -0,0 +1,231 @@
+package openlr.map.simplemockdb;
+
+import com.vividsolutions.jts.geom.*;
+import openlr.map.Line;
+import openlr.map.Node;
+import openlr.map.GeoCoordinates;
+import openlr.map.FormOfWay;
+import openlr.map.FunctionalRoadClass;
+import openlr.map.GeoCoordinatesImpl;
+import openlr.map.InvalidMapDataException;
+import org.geotools.geometry.jts.JTS;
+import org.geotools.geometry.jts.JTSFactoryFinder;
+import org.geotools.referencing.CRS;
+import org.geotools.referencing.crs.DefaultGeocentricCRS;
+import org.geotools.referencing.crs.DefaultGeographicCRS;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.operation.MathTransform;
+import org.opengis.referencing.operation.TransformException;
+
+
+import java.awt.geom.Point2D;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class SimpleMockedLine implements Line {
+
+    private generated.Line xmlLine;
```

Comment:
I think `generated` is a bad package name.  I think `openlr.map.simplemockdb.schema` would be better.  The JAXB plugin allows you to set a package name. https://www.mojohaus.org/jaxb2-maven-plugin/Documentation/v2.2/xjc-mojo.html#packageName

Reply:
9921bc46eaef1f2b022379c9e24cba52db5b1735